ui/packages/app/src/components/Dropdown/Dropdown.svelte

```diff
@@ -18,6 +18,12 @@
 	export let loading_status: LoadingStatus;
 
 	export let mode: "static" | "dynamic";
+
+	if (multiselect && !value) {
+		value = [];
+	} else {
```

Comment:
Should this be
```suggestion
	} else if (!value) {
```
to avoid overwriting a provided value to dropdown?

Reply:
Yeah defo. Its broken in more ways than just that atm tho, just making some changes.